solidity-frontend/src/main/java/org/sonarsource/solidity/frontend/Utils.java

```diff
@@ -13,15 +13,14 @@ public static String trimQuotes(String value) {
     return value.substring(1, value.length() - 1);
   }
 
-  // ... parser
   public static SolidityParser returnParserUnitFromParsedFile(String file) {
     CharStream cs = CharStreams.fromString(file);
     SolidityLexer sl = new SolidityLexer(cs);
-    // TokenStream tokens = new CommonTokenStream(sl);
+    // could use this instead: TokenStream tokens = new CommonTokenStream(sl);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: This block of commented-out lines of code should be removed. ([squid:CommentedOutCodeLine](https://sonarcloud.io/organizations/stylianos-agapiou-sonarsource-gi/rules#rule_key=squid%3ACommentedOutCodeLine))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=org.sonarsource.solidity%3Asonar-solidity&pullRequest=19&issues=AWQYrdldbXXTsx7O9fil&open=AWQYrdldbXXTsx7O9fil)

[ISSUE_KEY]: AWQYrdldbXXTsx7O9fil